nyt-top-20-science-news/.gitignore

```diff
@@ -0,0 +1,26 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
```

Comment:
this definitely seems like default boilerplate, but, given the inclusion of MS-specific library files above, do you prefer Mac or Windows (or Linux) for your work OS (I promise I'll try not to stereotype you regardless of your answer)?

Reply:
I prefer macOS. It makes full-stack development a bit easier. I've used windows for node and/or React and I've reached a point where I had to use a Linux VM to do the development because some commands would always break in Windows. I used Linux in college but it's not as user-friendly. Feels a bit clunky at times.

Reply:
phew